### ISSUE TITLE
Google Chat Card Layout

### ISSUE THREAD

**Issue by wernerb90** [gbandsmith/bitbucket-chat-bot#5]
Haai,

Not sure if this is something you can change with the "card" layout as opposed to just using a "message" ... and might be more a hangouts chat question - but is it possible to have a more horizontal layout for the card?  It takes A LOT of vertical space, where we have so much horizontal space available in the typical desktop chat window...

![image](https://user-images.githubusercontent.com/6905546/52695240-a2781000-2f74-11e9-9c29-a901f347f3bd.png)

**Comment by julbrs** [comment]
Exact, this is the "card" layout described here https://developers.google.com/hangouts/chat/reference/message-formats/cards

Maybe we can imagine a text layout also ? For sure it will take less space,
